src/Core/tests/Unit/Credentials/DateFromResultTest.php

```diff
@@ -32,23 +32,6 @@ public function testWithValidDate()
         self::assertEquals($time, $output->format('Y-m-d H:i:s'));
     }
 
-    public function testWithNoResponse()
-    {
-        $response = $this->getMockBuilder(Response::class)
-            ->disableOriginalConstructor()
-            ->onlyMethods(['info', 'resolve'])
-            ->getMock();
-
-        $response->expects(self::once())
-            ->method('info')
-            ->willReturn([]);
```

Comment:
This is not possible:
- current implementation always return an array with a response inside
- the class is internal
- the docbloc says it must return a response